packages/connector-types/src/infrastructure/httpServer/IHttpServer.ts

```diff
@@ -0,0 +1,11 @@
+import { RequestHandler } from "express";
+import { ConnectorInfrastructure } from "../ConnectorInfrastructure";
+import { HttpMethod } from "./HttpMethod";
+
+export interface IHttpServer extends ConnectorInfrastructure {
+    start(): Promise<void>;
+    stop(): void;
```

Comment:
I don't know if it's a good idea to expose these two methods on the same interface as the methods for configuring the http server.
But on the other hand, for now you probably only install modules that you develop yourself. So if you screw up and call `stop` accidentally, it's your own fault.